capstone/src/main/webapp/test files/home.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="https://apis.google.com/js/platform.js" async defer></script>
+        <script src="https://apis.google.com/js/api.js"></script>
+
+        <script src="script.js"></script>
+
+        <meta name="google-signin-client_id" content="">
+    </head>
+    <body>
+    <div id="content">
+        <div class="g-signin2" data-onsuccess="onSignIn"></div>
```

Comment:
Is there a reason the class is named "xxx-2"?

Reply:
See below.